code/modules/mob/living/carbon/human/human.dm

```diff
@@ -5,6 +5,7 @@
 	icon = 'icons/mob/human.dmi'
 	icon_state = "caucasian_m"
 	appearance_flags = KEEP_TOGETHER|TILE_BOUND|PIXEL_SCALE
+	var/has_agnosia_image = TRUE

```

Comment:
This is also weird

Reply:
To avoid dummies having it, although it's not strictly necessary

Reply:
Just have dummies override `generate_agnosia_image()` with a proc that does nothing.